buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -0,0 +1,80 @@
+import kotlin.String
+
+/**
+ * Generated by https://github.com/jmfayard/buildSrcVersions
+ *
+ * Update this file with
+ *   `$ ./gradlew buildSrcVersions`
+ */
+object Deps {
+    const val de_fayard_buildsrcversions_gradle_plugin: String =
+            "de.fayard.buildSrcVersions:de.fayard.buildSrcVersions.gradle.plugin:0.6.2"
+
+    const val androidx_test_core_ktx: String = "androidx.test:core-ktx:1.1.0"
+
+    const val kotlinx_coroutines_android: String =
+            "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1"
```

Comment:
Why are you adding this file? I thought the versions only mode only used `.properties` files.

Reply:
You are totally right. 
I was too much focused on fixing a nasty bug that I forgot what I was supposed to do. I will close this PR and send a new one :)